pom.xml

```diff
@@ -85,6 +85,16 @@
             <version>1.7.12</version>
         </dependency>
         <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.1.11</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.1.11</version>
+        </dependency>
+        <dependency>
```

Comment:
@FrankGasparovic will this cause any issues with slf4j  for apps which use log4j ?